sources/api/apiclient/src/main.rs

```diff
@@ -322,7 +322,6 @@ async fn run() -> Result<()> {
     let (args, subcommand) = parse_args(env::args());
     trace!("Parsed args for subcommand {:?}: {:?}", subcommand, args);
 
-    // We use TerminalMode::Stderr because apiclient users expect server response data on stdout.
     SimpleLogger::init(
```

Comment:
The old comments would no longer be accurate because they refer to a `TerminalMode` enum variant you're removing, but the purpose of the comments is still important.  We want to reinforce that errors go to stderr and anything less goes to stdout.  It helps us understand the functionality here, and also understand what's going on in the systemd service units that set up terminal log handling.

This one in particular is also special because apiclient has a different output story than the other binaries.

Can you please bring bring back the log messages, just correcting the `TerminalMode::Mixed` reference to `SimpleLogger`?

Reply:
Sure thing. Will revert.